swim/ping_request_sender.go

```diff
@@ -112,6 +115,29 @@ func (p *pingRequestSender) MakeCall(ctx json.Context, res *pingResponse) <-chan
 	return errC
 }
 
+// indirectPing is used to check if a target node can be reached indirectly.
+// The indirectPing is performed by sending a specifiable amount of ping
+// requests nodes in n's membership.
+func indirectPing(n *Node, target string, amount int, timeout time.Duration) (reached bool, errs []error) {
```

Comment:
I think it increases the readability since one might think the first return value could mean isHealthy.
isHealthy has a different meaning than reached when the indirectPing is inconclusive.

I think the renaming of this function from indirectHealthCheck to indirectPing, has helped the interpretability of return value, so I'm okay with removing it. 

I'd still like your thoughts on why we benefit from removing the names though.